tools/rosbag/src/play.cpp

```diff
@@ -61,7 +61,8 @@ rosbag::PlayerOptions parseOptions(int argc, char** argv) {
       ("try-future-version", "still try to open a bag file, even if the version is not known to the player")
       ("topics", po::value< std::vector<std::string> >()->multitoken(), "topics to play back")
       ("pause-topics", po::value< std::vector<std::string> >()->multitoken(), "topics to pause playback on")
-      ("bags", po::value< std::vector<std::string> >(), "bag files to play back from");
+      ("bags", po::value< std::vector<std::string> >(), "bag files to play back from")
+      ("wait-for-subscribers", "wait for subscribers before publishing");
```

Comment:
Maybe update the text to `wait for at least one subscriber on each topic before publishing` to make the condition clearer to the user.

Same for the Python help text.